build.gradle.kts

```diff
@@ -26,10 +27,24 @@ buildscript {
 
 plugins {
   id("com.github.gmazzo.buildconfig") version "2.1.0" apply false
+  alias(libs.plugins.spotless)
 }
 
 apply(plugin = "com.vanniktech.maven.publish.base")
 
+configure<SpotlessExtension> {
+  kotlin {
+    target("**/*.kt")
+    ktlint()
+      .editorConfigOverride(
```

Comment:
The fact the Spotless doesn't read editorconfig is ridiculous.

I have started switching projects to kotlinter which ensures you can centralize all configuration in editorconfig and it will be shared by the IDE and autoformatter without special configuration. It's worth considering here as well. Here's an example: https://github.com/cashapp/molecule/commit/aee2271c8f287658c818bbd5e0f7d5520a5c620d

Reply:
Yeah, we should just paste molecule’s here.

Reply:
Copy-pasted `.editorconfig` from molecule and converted the whole thing over to kotlinter. It's a bit sad to give up on Spotless as it's a more generic formatting tool over kotlinter, but a centralized `.editorconfig` is a _huge_ plus over (the possibility of) generic formatting using a single tool.

Reply:
This concern is obsolete with [Spotless Gradle Plugin 6.13.0](https://github.com/diffplug/spotless/blob/main/plugin-gradle/CHANGES.md#added). Reverting to use Spotless over Kotlinter.